lib/tasks/import.rake

```diff
@@ -0,0 +1,142 @@
+require 'csv'
+
+desc "Import Data"
+task import: :environment do
+  total_transactions = Transaction.all.length
+  Transaction.destroy_all
+  puts "#{total_transactions} transactions destroyed."
```

Comment:
Because this rake task can take up to 3 minutes, I've added statements so a dev isn't staring at a stagnant terminal, unsure of what's happening.